lib/matplotlib/tests/test_mathtext.py

```diff
@@ -12,7 +12,7 @@
 from matplotlib import mathtext
 
 math_tests = [
-    r'$a+b+\dots+\dot{s}+\ldots$',
+    r'$a+b+\dot s+\dot{s}+\ldots$',
```

Comment:
I'm pretty sure this case is intended to test that \dots produces three dots and \dot{s} produces an s with a dot over it.